pkg/storaged/client.js

```diff
@@ -393,13 +382,12 @@ function init_model(callback) {
                 var defer = cockpit.defer();
                 client.manager_lvm2 = proxy("Manager.LVM2", "Manager");
                 client.manager_iscsi = proxy("Manager.ISCSI.Initiator", "Manager");
-                wait_all([client.manager_lvm2, client.manager_iscsi],
-                         function () {
-                             client.features.lvm2 = client.manager_lvm2.valid;
-                             client.features.iscsi = (client.manager_iscsi.valid &&
+                Promise.allSettled([client.manager_lvm2.wait(), client.manager_iscsi.wait()]).then(() => {
```

Comment:
Checked that we can [use allSettled](https://caniuse.com/?search=Promise.allSettled) with our current babel browser settings.